src/app/[locale]/login/createAccount.tsx

```diff
@@ -106,7 +107,7 @@ export const CreateAccount = () => {
           annotation={dirty ? passwordError : null}
         />
         <TextField
-          label={<RequiredLabel label={t("login.confirm password")} />}
+          label={<RequiredLabel label={t('login.confirm password')} />}
```

Comment:
Hmm.... all these quote changes... can we check the VSCode settings and make sure the formatter is configured to prettier? I'm seeing everything in the codebase default to double quotes currently.